css/properties/background-clip.json

```diff
@@ -84,6 +84,7 @@
             ],
             "safari": {
               "version_added": "3",
+              "prefix": "-webkit-",
               "notes": "Webkit also supports the prefixed version of this property, and in that case, in addition to the current keywords, the alternative synonyms are: <code>padding</code>, <code>border</code>, and <code>content</code>."
```

Comment:
This note suggests that there is both a prefixed and unprefixed version of this. So, we need both and this note (slightly worded differently) for the prefixed version.

Reply:
I also found that this note is included in the Firefox data, despite that Firefox was never powered by WebKit...  (And in version 49, too!)  We'll probably want to look into this one more.

Reply:
I also found that this note is included in the Firefox data, despite that Firefox was never powered by WebKit...  (And in version 49, too!)  We'll probably want to look into this one more.

This also applies to `background-origin`.  I'm thinking that we should look into this one in a separate PR.

Reply:
Submitted as #4072.

Reply:
Thanks!